spec/doorkeeper-jwt/doorkeeper-jwt_spec.rb

```diff
@@ -113,7 +113,7 @@
 
       token = Doorkeeper::JWT.generate({})
       secret_key = OpenSSL::PKey::RSA.new File.read("spec/support/1024key.pem")
-      decoded_token = ::JWT.decode(token, secret_key, "RS512")
+      decoded_token = ::JWT.decode(token, secret_key, true, {algorithm: "RS512"})
```

Comment:
Line is too long. [81/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.